src/java.desktop/macosx/classes/sun/lwawt/macosx/CPlatformWindow.java

```diff
@@ -598,6 +600,11 @@ public void setMaximizedBounds(int x, int y, int w, int h) {
         execute(ptr -> nativeSetNSWindowStandardFrame(ptr, x, y, w, h));
     }
 
+    public void setAllowAutomaticWindowTabbing() {
+        boolean allowAutomaticWindowTabbing = Boolean.parseBoolean(System.getProperty("jdk.allowTabbedWindows"));
+        nativeSetAllowAutomaticTabbingProperty(allowAutomaticWindowTabbing);
+    }
+
```

Comment:
So we need a doPrivilged around the call to getProperty - in the new location of the static block

Reply:
Done